structs.h

```diff
@@ -347,13 +347,13 @@ typedef struct PlayerStruct {
 	unsigned char pBattleNet;
 #endif
 	BOOLEAN pManaShield;
-	char bReserved[3];
+	char bReserved[3]; //Unused

```

Comment:
They are unused and reserved so that in future releases they can be used in bugfixes without breaking the save game format.